en/docs/reference/pricing.md

```diff
@@ -2,229 +2,150 @@
-- <a class="pricing" id="reservationsAPI">Reservations API</a>: An external-facing API in Choreo that is used by client applications (web apps, mobile apps) to make flight bookings.
-- <a class="pricing" id="twillioAPI">Twilio API</a>: A third-party API that is used by the Reservations API in Choreo to send SMS notifications to customers when a booking has been made, canceled, or updated.
-- <a class="pricing" id="executionTime">Execution time</a>: The execution time taken for a process is the time taken between sending a request and receiving a response.
-<br/>
-<br/>
-
-#### Searching for a flight
-
-![Search for a flight](../assets/img/references/rest-api-example1.png)
-
-Let’s see how steps are calculated for the search process: 
-
-- As illustrated above, the client initiates a search for available flights through the web application. 
-- The web application then makes **1 API call to the [Search API](#searchAPI)** in Choreo. This is counted as **1 step**.
-- The search API in Choreo then makes **1 call to the Flights Database** to retrieve the information according to the search criteria entered by the client. This is counted as **1 step**. 
-
-- **Assuming the entire [execution time](#executionTime) for the search process takes less than 500ms** (starting from the web application making a call to Choreo, and Choreo making a call to the database), the search process thereby results in a **total of 2 steps**.
-
-<br/>
-
-#### Reserving a flight
-
-![Reserving a flight](../assets/img/references/rest-api-example2.png)
-
-Let’s see how steps are calculated for the reservation process:
-
-- As illustrated above, the client initiates the reservation process through the web application.
-- The web application proceeds to make **1 API call to the [Reservations API](#reservationsAPI)**. This is counted as 1 step.
-- The Reservation API then makes **1 API call to the [Bookings API](#bookingsAPI)**. This is counted as **1 step**. 
-- The bookings API makes **1 API call to the Booking Database** to update it with the booking details. This is counted as **1 step**.
-- Upon receiving the booking confirmation, the Reservation API makes **1 call to the [Twilio API](#twillioAPI)** to send an SMS confirmation to the customer. This is counted as **1 step**.
+## Plans
 
-- **Assuming the entire execution time for the reservation process takes 1200ms (1.2s)** , in addition to the above steps,  the total number of extra computation units consumed can be calculated as follows using ***floor((t)/500)) where t is the execution time***: 
+Choreo currently supports the following payment plans:
 
-          floor(1200/500) = 2 units = 2 steps
+**Free**
 
-Therefore the total number of steps for the reservation process is: 
+The free tier allows you to create up to five free components. These five components can be any combination of component types.
```

Comment:
Shall we rephrase this to: Allows you to create any five components.

Reply:
Doesn't that sound vague?

Reply:
I am sorry, can you elaborate?

Reply:
We might as well say it allows you to create 5 components and stop there. Just adding the word "any" does not value

The questions that we are trying to answer here can be "Should the components be of a specific type?" "Should all components have the same type?", or "Is this quota per type". The sentence "Allows you to create any five components." does not give any context for the user to relate such a statement to such questions.